### PR TITLE
[Comments] OT245-81- Endpoint creation Comments

### PR DESCRIPTION
**Pull request by CamiloLonM**
[https://alkemy-labs.atlassian.net/browse/OT245-81](url)

**Description**

AS user
I WANT to register new Comments
TO add them to an existing post

**Criteria of acceptance:**

POST /comments - You will need to validate that the news_id, user_id and body fields are sent.

**Empty body**
Cuerpo vacio

**wrong field**
Datos campo erroneo

**Error New y User**
error user

image

**OK!!**
post OK

### CODE REVIEW

**Comment by CamiloLonM** [comment]
El eslint no permite pasar las userId por User_id , y newId por news_Id como lo pide la cart.
eslint problema al destructuración
.

File: schemas/comments.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    notEmpty: {
+      errorMessage: 'body is empty',
+    },
+    isString: {
+      errorMessage: 'The body must be a integer',
+    },
+  },
+  userId: {
+    exists: {
+      errorMessage: 'user_id cannot be null',
```
Comment (R-Berrocal) [inline schemas/comments.js]: userId cannot be null

File: schemas/comments.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        if (!commentUser) {
+          throw new ErrorObject('UserId not Found', 400)
+        }
+        return userId
+      },
+    },
+  },
+  newId: {
+    exists: {
+      errorMessage: 'new_id cannot be null',
```
Comment (R-Berrocal) [inline schemas/comments.js]: newId cannot be null

**Review by R-Berrocal — CHANGES_REQUESTED** [review]

**Review by gabodmc — APPROVED** [review]
Solo corregir los typos que menciona Roberto

**Review by marattagian — APPROVED** [review]

**Review by R-Berrocal — APPROVED** [review]
Buena!

**Pull request merged**